.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,24 @@
+name: TypeScript Unit Tests
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - ready_for_review
+      - synchronize
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+      - name: Install Dependencies
+        run: npm install
+      - name: Run lint - prettier
+        run: npm run lint:prettier
```

Comment:
Any reason you're doing `npm run lint:prettier` and not just `npm run lint`

Reply:
Copied it from common - this is a good spot, updated